client/src/components/utilities.test.ts

```diff
@@ -22,6 +27,11 @@ afterEach(() => {
 })
 
 describe('utilities.ts', () => {
+  describe('tryJson', () => {
+    it('returns empty object if parse fails', () => {
+      expect(tryJson('{')).toEqual({})
+    })
+  })
```

Comment:
possible to test this condition by testing `api`? i think `tryJson` could remain a private implementation detail

Reply:
@jonahkagan Fair enough, when you put it that way. I mostly just flipped a coin in my head for which way I was going to test it, haha.